pkgs/applications/networking/instant-messengers/discord/openasar.nix

```diff
@@ -0,0 +1,63 @@
+{ lib, stdenv, fetchFromGitHub, fetchYarnDeps, nodejs-16_x, yarn, bash, fixup_yarn_lock }:
+
+stdenv.mkDerivation rec {
```

Comment:
Now re-reading this derivation, I really don't like the fact that we are implementing (yet another) way to build a NPM package.

Can't we use https://github.com/NixOS/nixpkgs/tree/master/pkgs/development/node-packages instead?

Reply:
`asar` was already there, I should have searched it better...